update-and-run.sh

```diff
@@ -18,7 +18,9 @@ fi
 
 if [ "$1" != "always_launch" ]; then
     if ! $cmd_prefix docker compose ps | grep "mantis" >/dev/null 2>&1; then
-        print_error "docker compose appears to be brought down. Will not proceed to avoid relaunching."
+        print_error "Sublime Platform appears to have been manually brought down. Will not proceed to avoid relaunching."
```

Comment:
Optional, just thought it sounded better.

```suggestion
        print_error "Sublime Platform appears to have been manually shut down. Will not proceed to avoid relaunching."
```